.github/workflows/autogpt-builder-ci.yml

```diff
@@ -17,6 +17,24 @@ defaults:
     working-directory: rnd/autogpt_builder
 
 jobs:
+  format:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '21'
+
+      - name: Install dependencies
+        run: |
+          npm install
+
+      - name: Run format
+        run: |
+          npm run format
```

Comment:
Run with —check https://prettier.io/docs/en/cli.html#--check

Reply:
Great call out here @ntindle, 

I have updated the ci to no longer add an additional job but instead run the check before linting as linting is generally more costly. 

If prettier was not ran before the push the job will fail. 

I'm not sure if we wanted an error message to be echoed here so I left it out. Please let me know if it is requested.